opensearch-operator/api/v1/opensearch_types.go

```diff
@@ -284,6 +284,9 @@ type ClusterStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:shortName=os;opensearch
 // Es is the Schema for the es API
+// +kubebuilder:printcolumn:name="version",type="string",JSONPath=".status.version",description="Elasticsearch version"
```

Comment:
Please change`Elasticsearch` to `Opensearch` as we only handle opensearch with this operator.

Reply:
done